socorro/external/rabbitmq/crashstorage.py

```diff
@@ -208,11 +210,20 @@ def _suppress_duplicate_jobs(self, crash_id, acknowledgement_token):
                 crash_id
             )
             # ack this
-            self.transaction(
-                self._transaction_ack_crash,
-                crash_id,
-                acknowledgement_token
-            )
+            try:
+                self._ack_crash(
+                    channel,
+                    crash_id,
+                    acknowledgement_token
+                )
+            except Exception:
+                self.config.logger.error(
+                    'RabbitMQCrashStorage tried to acknowledge duplicate crash %s'
+                    ', but failed.',
+                    crash_id,
+                    exc_info=True
+                )
```

Comment:
I don't understand the why you took this out of the transaction. What're you trying to fix here?